pkg/systemd/service-details.scss

```diff
@@ -33,6 +33,14 @@
   margin-right: 0.5em;
 }
 
+.service-top-panel {
+  flex-wrap: nowrap;
```

Comment:
Please allow the header to wrap - squeezing it all in one line is not what we want. 

I will let @garrett approve but I would expect the actions to go to next line but stay on the right side in mobile mode.

![Screen Shot 2023-05-03 at 13 20 27](https://user-images.githubusercontent.com/14921356/235902260-658ba41d-50a2-4d0e-83f2-7f34c9285484.png)